sdk/core/azure-core/test/performance/inc/azure/core/test/performance/nullable.hpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Test the Nullable component performance.
+ *
+ */
+
+#pragma once
+
+#include <azure/performance_framework.hpp>
+
+#include <memory>
+
+namespace Azure { namespace Core { namespace Test { namespace Performance {
```

Comment:
NIT, namespace Azure::Performance::Core

Reply:
I will create a new issue to define the test file structure and namespace that we want to follow.
As right now we are not doing Azure::Test::  or Azure::Performance and making this change only for this PR wouldn't be ideal.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1694